---

# Gamma Distribution

A call center receives calls according to a Poisson process with rate $\lambda = 12$ per hour.  
Let $T_4$ be the waiting time until the 4th call.

Since $T_4 \sim \text{Gamma}(k=4, \theta = 1/12)$, compute:

1. $P(T_4 \le 10 \text{ minutes})$  
2. $P(T_4 > 20 \text{ minutes})$

In [3]:
import scipy.stats as stats
alpha = 4
theta = 5

ans1 = stats.gamma.cdf(10, a=alpha, scale=theta)
print(f"1) {ans1}")

ans2 = stats.gamma.sf(20, a=alpha, scale=theta)
print(f"2) {ans2}")

1) 0.14287653950145296
2) 0.43347012036670896


1) There's about a 14.3% chance that 4 calls will arrive within the first 10 minutes.
2) So there’s about a 43.3% chance that it takes more than 20 minutes for four calls to arrive.